app/src/main/java/com/cornellappdev/uplift/util/TestingConstants.kt

```diff
@@ -1,35 +1,83 @@
 package com.cornellappdev.uplift.util
 
+import androidx.compose.runtime.mutableStateListOf
+import com.cornellappdev.uplift.R
 import com.cornellappdev.uplift.models.*
 import java.util.*
 
 val exCalendarMarch25: Calendar = GregorianCalendar(2023, 2, 25)
 
+val exampleClassMusclePump6 = UpliftClass(
```

Comment:
Due to being connected to the backend, it seems you can delete the example data you made now